### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by manuelmazzuola** [rtgibbons/grunt-cloudfiles#29]
Is this dead?

**Comment by rtgibbons** [comment]
I don't use it anymore, we've moved on to using S3 for things, and using the aws cli for uploading files vs a grunt task.

I know there are still people using it. But I don't have the time to innovate on it. I believe @kenperkins still has an eye on it, but there hasn't been any work done on it lately. I'll had a call to action for some help on the README

**Comment by kenperkins** [comment]
That's correct; I need to go through and update this to the latest pkgcloud version.

**Comment by kenperkins** [comment]
Now that I've finally gotten pkgcloud 1.0 out the door, I plan to update this asap. As in starting next week.
